test/testcafe/spec/DeviceProbingChromeDTC_spec.js

```diff
@@ -0,0 +1,118 @@
+import { RequestMock } from 'testcafe';
+import { checkA11y } from '../framework/a11y';
+import DeviceChallengePollPageObject from '../framework/page-objects/DeviceChallengePollPageObject';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import identify from '../../../playground/mocks/data/idp/idx/identify';
+import deviceProbingChromeDTC from '../../../playground/mocks/data/idp/idx/device-probing-chrome-dtc';
+import identifyWithDeviceProbingLoopback from '../../../playground/mocks/data/idp/idx/identify-with-device-probing-loopback';
+
+const mockChromeProbingView = RequestMock()
```

Comment:
Tests here are not complete end 2 end because device signals are not used in current backend implementation. Will create a jira to update the tests once the policy part (evaluate device signals based on ASOP) is done.